addons/website_event_track_live/views/event_track_templates_list.xml

```diff
@@ -1,19 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<template id="tracks_display_list" inherit_id="website_event_track.tracks_display_list">
+<template id="track_list_item" inherit_id="website_event_track.track_list_item">
     <!-- TRACK LIST: ADD REPLAY TAG FOR FINISHED TRACKS -->
-    <xpath expr="//div[@t-foreach='tracks']//div[hasclass('col-12')]//span[@t-elif='not track.is_track_done and not track.is_track_soon']" position="after">
+    <xpath expr="//div[hasclass('col-12')]//span[@t-elif='not track.is_track_done and not track.is_track_soon']" position="after">
         <a t-elif="track.youtube_video_url and (track.is_published or is_event_user)"
             t-att-href="track.website_url" class="badge text-bg-danger">Replay
         </a>
     </xpath>
     <!-- ADD YOUTUBE ICON -->
-    <xpath expr="//div[@t-foreach='tracks']//div[hasclass('col-12')]//a/span[@t-field='track.name']" position="before">
-        <i t-if="track.date and track.youtube_video_url and (track.is_track_soon or track.is_track_live or track.is_youtube_replay)"
-            class="fa fa-youtube-play text-danger me-1"/>
-    </xpath>
-    <xpath expr="//div[@t-foreach='tracks']//div[hasclass('col-12')]//t/span[@t-field='track.name']" position="before">
+    <xpath expr="//div[hasclass('col-12')]//t[@t-field='track.name']" position="before">
```

Comment:
`Element '<xpath expr="//div[hasclass(&#39;col-12&#39;)]//t[@t-field=&#39;track.name&#39;]">' cannot be located in parent view`

Arf... The selector does not work. May I ask you to fix it?